### PR TITLE
Add introduction video for athene2-editor

### LINKED ISSUE #229: Include the explanation video for the editor once
* Include the explanation video, https://de.serlo.org/entity/view/47257 in the editor
* include this only once:
  * For example, similar to many applications: checkbox, whether the video should be shown in the future
  * Or: really just once (set flag)

### PR DESCRIPTION
**Pull request by inyono**
Close #229.

Todo:
- [x] Make the modal larger
- [x] Move JS in athene2 / athene2-editor?
- [x] Translate strings

screen shot 2015-12-01 at 12 29 29

### CODE REVIEW

File: src/module/Ui/templates/editor/partials/video-modal.phtml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<div class="modal fade" id="editorIntroduction" tabindex="-1" role="dialog" aria-labelledby="editorIntroductionLabel">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            </div>
+            <div class="modal-body">
+                <p class="embed-responsive embed-responsive-16by9">
+                    <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/l7WknEvcVQk?html5=1&hl=<?php echo $this->currentLanguage(); ?>&showinfo=0"></iframe>
```
Comment (arekkas) [inline src/module/Ui/templates/editor/partials/video-modal.phtml]: encode

**Comment by arekkas** [comment]
no need to move to athene2-editor!

**Comment by arekkas** [comment]
makes more sense here

**Comment by inyono** [comment]
Also leaving it inline, since it is only relevant for this specific page? And what about styles? Inline / athene2 SCSS / athene2-editor SCSS?

**Comment by inyono** [comment]
out

**Comment by arekkas** [comment]
looks awesome! tell me when good to merge

**Comment by inyono** [comment]
see question above :)

**Comment by arekkas** [comment]
hm no keep things sane, let's move it to a dedicated file, even if it's a one time use case

**Comment by inyono** [comment]
done

**Comment by arekkas** [comment]
LGTM, feel free to merge yourself

**Pull request merged**